Gems/Atom/RHI/Code/Include/Atom/RHI/DrawItem.h

```diff
@@ -130,10 +130,7 @@ namespace AZ::RHI
 
         union
         {
-            struct
-            {
-                bool m_enabled : 1;    // Whether the Draw Item should render
-            };
+            bool m_enabled = { 1 }; // Whether the Draw Item should render
             uint8_t m_allFlags;
```

Comment:
I think we still want to keep how the union is defined, which allows you expand the flag list without increase the structure's size.

Reply:
Also, using previous declaration, user may use m_allFlags to control the m_enabled's value too. With this change, this is no meaning to keep m_allFlags anymore.

Reply:
That was fancy union'ing, but didn't work.
The idea being users could set flags or check if enabled (which ever was more readable).
Any objections to:
1.  Only keeping m_allFlags and keeping a comment that having no bits set will be interpreted as disabled?
2. Keeping both m_enabled and m_allFlags but both as uint8_t

Reply:
Note: m_allFlags variable is never read.

Reply:
https://github.com/o3de/o3de/pull/17024 should address this issue. We can close this PR.